src/sql/workbench/services/accountManagement/browser/accountListRenderer.ts

```diff
@@ -64,8 +64,8 @@ export class AccountPickerListRenderer implements IListRenderer<azdata.Account,
 		DOM.append(tableTemplate.icon, badge);
 		tableTemplate.badgeContent = DOM.append(badge, DOM.$('div.badge-content'));
 		tableTemplate.label = DOM.append(tableTemplate.root, DOM.$('div.label'));
-		tableTemplate.contextualDisplayName = DOM.append(tableTemplate.label, DOM.$('div.contextual-display-name'));
-		tableTemplate.displayName = DOM.append(tableTemplate.label, DOM.$('div.display-name'));
+		tableTemplate.contextualDisplayName = DOM.append(tableTemplate.label, DOM.$('h1.contextual-display-name'));
```

Comment:
I am afraid this is not the correct of using headings, as shown in the screenshot below, 'Microsoft Corp' and 'Alan Ren - alanren@microsoft.com' are just the properties of this account, they might have different font size, but not a hierarchy structure. I've asked the tester who opened the issue to provide more information, please follow up with her.